package.json

```diff
@@ -3,28 +3,18 @@
   "author": "Nomic Foundation",
   "license": "SEE LICENSE IN EACH PACKAGE'S LICENSE FILE",
   "private": true,
-  "workspaces": {
-    "packages": [
-      "packages/core",
-      "packages/ui",
-      "packages/hardhat-plugin",
-      "packages/hardhat-plugin-ethers",
-      "packages/hardhat-plugin-viem",
-      "examples/*"
-    ]
-  },
   "scripts": {
-    "build": "npm run build -w packages",
-    "lint": "npm run lint --workspaces",
+    "build": "pnpm --filter \"./packages/*\" build",
```

Comment:
In Harhdat the equivalent command is `pnpm run --recursive build` combined with a `pnpm-workspace.yaml` file.
Is there a tradeoff here? Is the goal to constrain down the build to only the packages subfolders?

Reply:
Ah, the first time I tried that version was before I added the `pnpm-workspace.yaml` file. I already had this version in place by the time I added that file, so I didn't connect the dots that that was the specific reason it wasn't working before. I don't think there's any tradeoffs, but I'll change it to match Hardhat's style for consistency